shared/js/content-scripts/element-hiding.js

```diff
@@ -45,7 +46,8 @@
                     window.top.postMessage({frameUrl: document.location.href, type: 'frameIdRequest'}, req.mainFrameUrl)
                 }
             } else if (req.type === 'blockedFrame') {
-                document.getElementsByTagName('iframe').forEach((frame) => {
+                this.foundFrames = document.getElementsByTagName('iframe')
+                Array.prototype.forEach.call(this.foundFrames, (frame) => {
```

Comment:
how about we replace `getElementsByTagName` with `querySelectorAll`? That way we will be able to use `.forEach`.